cmd/prometheus/main_test.go

```diff
@@ -155,3 +156,20 @@ func TestComputeExternalURL(t *testing.T) {
 		}
 	}
 }
+
+// Provide an invalid configuration file and verify the exit status indicates the error
```

Comment:
Comments should be full sentences and end in a full stop.

Reply:
This is actually something that even varies across the Prometheus team, I wouldn't worry about it - especially for a tiny PR like this.